ingest/Snakefile

```diff
@@ -388,17 +410,33 @@ rule heuristic_group_segments:
     """Group segments based on heuristic, join with previous groups if available"""
     input:
         script="scripts/heuristic_group_segments.py",
-        metadata="results/metadata_ungrouped.ndjson" if GROUPS_OVERRIDE_JSON else "results/metadata_post_prepare.ndjson",
-        sequences="results/sequences_ungrouped.ndjson" if GROUPS_OVERRIDE_JSON else "results/sequences.ndjson",
-        metadata_grouped="results/metadata_grouped.ndjson" if GROUPS_OVERRIDE_JSON else "results/metadata_post_prepare.ndjson",
-        sequences_grouped="results/sequences_grouped.ndjson" if GROUPS_OVERRIDE_JSON else "results/sequences.ndjson",
+        metadata=(
```

Comment:
sorry this a formatting change

Reply:
there are a bunch more, would be great to format once, make PR, we'll merge, then rebase onto that